vcr/patch.py

```diff
@@ -229,6 +194,40 @@ def _boto(self):
         else:
             from .stubs.boto_stubs import VCRCertValidatingHTTPSConnection
             yield cpool, 'CertValidatingHTTPSConnection', VCRCertValidatingHTTPSConnection
+            
+    def _mock_urllib3_triples(self, cpool, stubs):
```

Comment:
This should probably be given a different name since it does not actually return triples, but context managers. Notice that though it does construct the triples internally, it applies `_build_patchers_from_mock_triples` at the end of the function definition.

Way to keep this DRY!